schemas/public/ekscluster-kfd-v1alpha2.json

```
@@ -308,21 +313,22 @@
     },
     "Spec.Infrastructure.Vpc.Network.SubnetsCidrs": {
       "type": "object",
+      "description": "Network CIDRS configuration for private and public subnets.",
       "additionalProperties": false,
       "properties": {
         "private": {
           "type": "array",
           "items": {
             "$ref": "#/$defs/Types.Cidr"
           },
-          "description": "These are the CIRDs for the private subnets, where the nodes, the pods, and the private load balancers will be created"
+          "description": "Network CIRDs for the private subnets, where the nodes, the pods, and the private load balancers will be created"
         },
         "public": {
           "type": "array",
           "items": {
             "$ref": "#/$defs/Types.Cidr"
           },
-          "description": "These are the CIDRs for the public subnets, where the public load balancers and the VPN servers will be created"
+          "description": "Network CIDRs for the public subnets, where the public load balancers and the VPN servers will be created"
```

```suggestion
          "description": "The network CIDRs for the public subnets, where the public load balancers and the VPN servers will be created"
```
Just to keep the same wording as the other fields